share/fsmon/monitor.go

```diff
@@ -575,8 +575,7 @@ func (w *FileWatch) getCoreFile(cid string, pid int, profile *share.CLUSFileMoni
 	dirList := make(map[string]*osutil.FileInfoExt)
 	singleFiles := make([]*osutil.FileInfoExt, 0)
 
-	// get files and dirs from all filters
-	for _, filter := range profile.Filters {
+	addFilterFn := func(filter share.CLUSFileMonitorFilter){
```

Comment:
This is cosmetic to reduce the copy paste code. No changes in the logic was done and will make it easier to make changes in the future.